app/views/language.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import generics
+
+from ..models import LanguageProficiency, SpokenLanguage
+from ..serializers import LanguageProficiencySerializer, SpokenLanguageSerializer
+
+
+class LanguageProficiencyList(generics.ListAPIView):
```

Comment:
Why did you opt for Generic views? 
Is this like a default view file? I see that all the classes do the same things

Reply:
Generic views is modeled like a template for requests, with APIView, I'd have had to write out the GET request function and I didn't think that was necessary since I don't need to customize the language proficiency. ListAPIView only handles get all objects request.